### DISCUSSION TITLE
推薦一款文字小說遊戲<活俠傳>

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#3709] [General]
活俠傳是一款由原始鳥熊開發出來的文字武俠冒險遊戲
其中玩家要扮演唐門最後一個外姓弟子----趙活
而主角的特色就是，長得有夠醜
故事的重點就是你不是大俠，只是一屆凡人，要怎麼在這個武林闖蕩

我自己在剛完的時候，其實對於武俠遊戲是沒概念的，但是隨著遊戲進行，我卻被深深的吸引進去作者創造的故事中
我覺得這個遊戲的最大特色是非常細膩的人物刻畫，非常真實劇情(當然不是指很科學的意思)
以及主角在故事中每天做的事情，每次的抉擇都會影響他的數值，而這些數值會直接攸關到在大事件的分歧點
而且作品的細節是，根據數值還有選擇，每個地方的文本都有些許不同，常常會發現之前沒看過的劇情

大家如果寒假沒事，可以去體驗一下這款國產的武俠遊戲

**Comment by kai1660** [comment]
剛玩過，非常推薦大家來玩，故事很有趣，又是國產的，一定要大力支持。
